clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/validators/BasicAmazonDeployDescriptionValidator.groovy

```diff
@@ -41,7 +42,7 @@ class BasicAmazonDeployDescriptionValidator extends AmazonDescriptionValidationS
     if (!description.credentials) {
       errors.rejectValue "credentials", "basicAmazonDeployDescription.credentials.empty"
     } else {
-      credentials = accountCredentialsProvider.getCredentials(description?.credentials?.name)
+      credentials = credentialsRepository.getOne(description?.credentials?.name)
       if (!(credentials instanceof AmazonCredentials)) {
```

Comment:
You can just check `credentials == null` since `NetflixAmazonCredentials` extends `AmazonCredentials`